src/test/java/com/darzalgames/darzalcommon/hexagon/HexagonDirectionTest.java

```diff
@@ -0,0 +1,94 @@
+package com.darzalgames.darzalcommon.hexagon;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.darzalgames.darzalcommon.data.Coordinate;
+
+class HexagonDirectionTest {
+
+	@Test
+	void getNeighborCoordinate_topLeft_hasCorrectCoordinates() {
+		int startQ = 5;
+		int startR = -3;
+
+		Coordinate coordinate = HexagonDirection.getNeighborCoordinate(startQ, startR, HexagonDirection.TOP_LEFT);
+
+		int expectedQ = startQ - 1;
```

Comment:
for expected results like this, they should be hardcoded instead of calculated. As well, when the expected values are simple, you can put them directly in the assertions, e.g. `assertEquals(4, coordinate.i);`